apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/ExplanationOfBenefitE2EBase.java

```diff
@@ -0,0 +1,1115 @@
+package gov.cms.bfd.server.war;
```

Comment:
This is 90% copying the V2 tests into a common location, and 10% creating the few test cases that existed in the v1 suite but not v2 suite. This class allows for all test cases to be shared across both versions, ultimately increasing our coverage for both.